### DISCUSSION TITLE
求火炬之光1代，以前的玩不了了

### DISCUSSION THREAD

**Discussion by zj185690007** [Gamuxorg/bbs#241] [投递和请求游戏]
游戏名称：火炬之光1
游戏名称(英文)：Torchlight
下载地址 :（如果有的话）：http://gamux.org/clientgame/torchlight/torchlight-20130529.sh
https://store.steampowered.com/app/41500/Torchlight/

链接: https://pan.baidu.com/s/1VM3uejVIlMGhWvwwpDdHCQ 提取码: f2tu 复制这段内容后打开百度网盘手机App，操作更方便哦

**Comment by shouhuanxiaoji** [comment]
已解决
